### PR TITLE
Add image model and return images on GET for full assessment

### PR DESCRIPTION
**Pull request by takkaria**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,13 +204,22 @@ Content-Type: application/json
...
+            "height": 300,
+            "thumbnail_url": "https://gallery.img/6_thumb.jpg",
+            "thumbnail_width": 200,
+            "thumbnail_height": 150,
+            "note": "",
+            "is_featured": true
+        }
+    ],
     "data": {
-        "master": {
```
Comment (paulfurley) [inline README.md]: not sure this example structure was meant to get deleted here?

File: mhep/mhep/dev/tests/views/test_retrieve_update_destroy_assessment.py [hunk 2]
```diff
@@ -80,6 +96,52 @@ def test_returns_404_for_bad_id(self):
         response = self.client.get(f"/{VERSION}/api/assessments/bad-id/")
         assert status.HTTP_404_NOT_FOUND == response.status_code
 
+    def test_image_featured(self):
```
Comment (paulfurley) [inline mhep/mhep/dev/tests/views/test_retrieve_update_destroy_assessment.py]:
```suggestion
    def test_structure_of_featured_and_normal_image(self):
```

File: mhep/mhep/dev/serializers.py [hunk 3]
```diff
@@ -25,6 +25,44 @@ def get_mdate(self, obj):
...
 
 
+class ImagesMixin():
+    def get_images(self, obj):
+        if obj.featured_image is not None:
+            featured_id = obj.featured_image.id
+        else:
+            featured_id = None
+
+        s = ImageSerializer(obj.images, context={"featured_id": featured_id}, many=True)
```
Comment (paulfurley) [inline mhep/mhep/dev/serializers.py]: clever!

File: README.md (hunk 1 above)
Comment (takkaria) [reply README.md]: Yes you're right - now reinstated

File: mhep/mhep/dev/serializers.py (hunk 3 above)
Comment (takkaria) [reply mhep/mhep/dev/serializers.py]: 😊

**Pull request merged**